checkov/cloudformation/runner.py

```diff
@@ -75,14 +76,40 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
                             variable_evaluations = {}
 
                             skipped_checks = ContextParser.collect_skip_comments(entity_code_lines)
-
-                            results = cfn_registry.scan(cf_file, {resource_name: resource}, skipped_checks,
+                            entity = {resource_name: resource}
+                            results = cfn_registry.scan(cf_file, entity, skipped_checks,
                                                         runner_filter)
+                            tags = Runner.get_resource_tags(entity)
                             for check, check_result in results.items():
                                 record = Record(check_id=check.id, check_name=check.name, check_result=check_result,
                                                 code_block=entity_code_lines, file_path=cf_file,
-                                                file_line_range=entity_lines_range,
-                                                resource=resource_id, evaluations=variable_evaluations,
-                                                check_class=check.__class__.__module__, file_abs_path=file_abs_path)
+                                                file_line_range=entity_lines_range, resource=resource_id,
+                                                evaluations=variable_evaluations,check_class=check.__class__.__module__,
+                                                file_abs_path=file_abs_path, entity_tags=tags)
                                 report.add_record(record=record)
-        return report
\ No newline at end of file
+        return report
+
+    @staticmethod
+    def get_resource_tags(entity):
```

Comment:
can we move both of those static methods to a new file?

Reply:
We could, but it would still be a CFN-specific logic, which is why I put them here. (In other words, getting CFN resource tags looks different from Terraform). And it seems to be like this is logic that is runner-centric.

Were you thinking just a utils file inside the CFN directory?